source/geod24/LocalRest.d

```diff
@@ -723,8 +726,9 @@ public final class RemoteAPI (API, alias S = VibeJSONSerializer!()) : API
 
         void runNode ()
         {
-            scope node = new Implementation(cargs);
+            is_remote_api = true;
             scheduler = new C.FiberScheduler;
+            scope node = new Implementation(cargs);
```

Comment:
No, I think right here before calling the ctor should work.

Reply:
But practically they are the same I guess. Let me give it a try.

Reply:
That still seems to segfault for me. Can you give this a shot? :)